godotproject/Player.gd

```diff
@@ -3,12 +3,23 @@ extends KinematicBody2D
 signal tongue_start
 signal tongue_stop
 
-export (int) var speed = 200
-export (int) var jump_speed = -500
-export (int) var gravity = 1200
+# Constants relating to horizontal movement
+export (float) var walk_horiz_accel = 1200
+export (float) var walk_speed_limit = 225
+export (float) var going_too_fast_slowdown_accel = walk_speed_limit / 0.25
+export (float) var friction_coeff = 1.5
+export (float) var ground_turn_coefficient = friction_coeff * 1.5
+export (float) var air_horiz_accel = walk_horiz_accel * 0.6
+export (float) var air_speed_limit = walk_speed_limit * 1.0
+export (float) var air_turn_coefficient = 1.2
+
+export (float) var jump_leniency_time = 5 / 60.0
+export (float) var jump_speed = 500
 export (float) var jump_length = 10 / 60.0
+export (float) var shorthop_speed = jump_speed * 0.6
 export (float) var shorthop_length = 6 / 60.0
-export (float) var shorthop_ratio = 0.6
+export (float) var gravity = 1200
+export (float) var terminal_velocity = 400
```

Comment:
this would prevent players going infinitely fast right?

Reply:
At the moment terminal_velocity is only applied in the downward direction, basically to stop the user from falling too quickly

Reply:
ahh ok. If we decide to let the player chain boosts it might be a good idea to add it to horizontal so the dont end up being able to pass through platforms